pages/verify-certification-score.vue

```diff
@@ -0,0 +1,234 @@
+              Votre code de vérification ne correspond pas au format: P-XXXXX
+            </p>
+          </div>
+          <div class="input-field">
+            <label for="score">{{ candidateScoreLabel }}</label>
+            <input
+              id="score"
+              v-model="score"
+              :class="hasScoreRegexError && getRegexErrorClass"
+              type="text"
+              required="true"
+              name="candidateScore"
+            />
+            <p v-if="hasScoreRegexError" class="error-paragraph">
+              Le score du candidat doit être composé de 3 chiffres
+            </p>
+          </div>
+          <button class="btn-primary" type="submit">
+            {{ buttonLabel }}
+          </button>
+        </form>
+        <div v-if="formSent" class="form-sent">
+          <fa :icon="fas.faExclamationTriangle" />
+          <span>Il n'y a pas de certificat Pix correspondant</span>
+        </div>
+      </div>
+    </main>
+  </div>
+</template>
+
+<script>
+import { fas } from '@fortawesome/free-solid-svg-icons'
+import { documentFetcher } from '~/services/document-fetcher'
+
+export default {
+  name: 'VerifyCertificatioScore',
+  nuxtI18n: {
+    paths: {
+      'fr-fr': '/verifier-score-certification',
+      'en-gb': '/verify-certification-score',
+    },
+  },
+  async asyncData({ app, error, req, currentPagePath }) {
+    try {
+      const document = await documentFetcher(
+        app.i18n,
+        req
+      ).getVerifyCertificationForm()
+      if (process.client) window.prismic.setupEditButton()
+
+      return {
+        currentPagePath,
+        meta: document.data.meta,
+        document: document.data,
+        documentId: document.id,
+      }
+    } catch (e) {
+      // eslint-disable-next-line no-console
+      console.error(e)
+      error({ statusCode: 404, message: 'Page not found' })
+    }
+  },
+  data() {
+    return {
+      score: '',
+      code: '',
+      errors: [],
+      codeRegex: /^P-[0-9]{5}$/i,
+      scoreRegex: /^[0-9]{1,3}$/,
+      hasCodeRegexError: false,
+      hasScoreRegexError: false,
+      formSent: false,
+    }
+  },
+  computed: {
+    buttonLabel() {
+      return this.document.button_label
+    },
+    title() {
+      return this.document.title
+    },
+    verificationCodeLabel() {
+      return this.document.code_de_verification
+    },
+    candidateScoreLabel() {
+      return this.document.score_declare_du_candidat
+    },
+    getRegexErrorClass() {
+      return 'regex-error'
+    },
+    fas() {
+      return fas
+    },
+  },
+  methods: {
+    submit(e) {
+      e.preventDefault()
+      if (this.checkForm()) {
+        this.formSent = true
+      }
+    },
+    checkForm() {
+      this.hasCodeRegexError = !this.testCodeRegex()
+      this.hasScoreRegexError = !this.testScoreRegex()
+
+      return !this.hasCodeRegexError && !this.hasScoreRegexError
+    },
+    testCodeRegex() {
```

Comment:
Vu que c'est un booléan on peut commencer le nom par un "is" ? Par ex : `isCodeCorrect` ?